src/libstd/sys_common/backtrace.rs

```diff
@@ -164,11 +183,27 @@ fn filter_frames(frames: &[Frame],
     let skipped_before = frames.iter().position(|frame| {
         is_good_frame(*frame, BAD_PREFIXES_TOP)
     }).unwrap_or(frames.len());
-    let skipped_after = frames[skipped_before..].iter().rev().position(|frame| {
+    let idx_catch_panic = skipped_before + frames[skipped_before..].iter().rposition(|frame| {
+        let mut is_rmcp = false;
+        let _ = resolve_symname(*frame, |symname| {
+            if let Some(mangled_symbol_name) = symname {
+                if mangled_symbol_name == "__rust_maybe_catch_panic" {
```

Comment:
This needs to be `mangled_symbol_name.contains("rust_maybe_catch_panic")` (regardless of other concerns) because the symbol can be named `panic_unwind::__rust_maybe_catch_panic` or `_rust_maybe_catch_panic` (on Windows).